packages/eslint-plugin-js/rules/indent/README.md

```diff
@@ -27,7 +27,7 @@ This rule enforces a consistent indentation style. The default style is `4 space
 
 This rule has a mixed option:
 
-For example, for 2-space indentation:
+For example, for 2-spaces indentation:
```

Comment:
I think with the dash it should be singular.

Reply:
https://english.stackexchange.com/questions/79838/why-are-certain-single-word-compound-nouns-pluralized-in-the-middle
https://english.stackexchange.com/questions/8450/words-that-are-pluralized-in-the-middle
https://ell.stackexchange.com/questions/134575/what-is-it-plural-with-s

Reply:
reverted